modules/runner/src/integrationTest/java/org/apache/ignite/internal/runner/app/ItTablesApiTest.java

```diff
@@ -322,12 +549,139 @@ synchronized void stopInhibit() {
      */
     protected Table createTable(Ignite node, String schemaName, String shortTableName) {
         return node.tables().createTable(
-                schemaName + "." + shortTableName, tblCh -> convert(SchemaBuilders.tableBuilder(schemaName, shortTableName).columns(
-                                SchemaBuilders.column("key", ColumnType.INT64).asNonNull().build(),
-                                SchemaBuilders.column("valInt", ColumnType.INT32).asNullable().build(),
-                                SchemaBuilders.column("valStr", ColumnType.string()).withDefaultValueExpression("default").build()
+                schemaName + "." + shortTableName,
```

Comment:
seems we already need _DROP TABLE [ IF EXISTS ]_ syntax

Reply:
It will be fixed it IGNITE-15926